tidy3d/plugins/autograd/README.md

```diff
@@ -152,6 +152,8 @@ Next on our roadmap (targeting 2.8 and 2.9, summer 2024) is to support:
 - `PoleResidue` and other dispersive models.
 - custom (spatially-dependent) dispersive models, allowing topology optimization with metals.
 
+- `ComplexPolySlab`
```

Comment:
remove this change? also, wondering if you ever updated this readme to reflect the field data operations (put them in the "supported" section?) if not, maybe we can do that in this PR? 

idea is to have this README contain the latest state of released, in progress, and todo.